### ISSUE TITLE
Eigengrau mode not working

### ISSUE THREAD

**Issue by Lazyme114** [LagradOst/CloudStream-3#106]
I have checked the eigengrau mode but the speed option is not showing. I have realme 3 pro. I have 2 version of this app beta version (1.7.*) this shows the speed option in this. But the latest version (1.9.*) doesn't show the speed option.

**Comment by LagradOst** [comment]
Already fixed, try pre release

**Comment by yephny** [comment]
@LagradOst I recommend releasing a hotfix for such fixed bugs. I downloaded the beta (pre-release) which is basically a different app that does not show my watchlist from the original app. For people like me who wish to follow only the main release with no possibly unstable or experimental features, it is best to release hotfixes with the main version instead. Now that I've switched to beta, I need to watch out for the main app releases so that I can switch to it when a new version is out just to get back the Eigengrau mode.
